src/main/java/Duke.java

```diff
@@ -1,10 +1,179 @@
+import duke.Deadline;
+import duke.Event;
+import duke.Todo;
+
+import java.util.ArrayList;
+import java.io.*;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws FileNotFoundException {
```

Comment:
hmmmm no point throwing an exception from main; there is no one to catch it!
you need to handle the exception here.